src/newWizard/Wizard.tsx

```diff
@@ -87,7 +87,7 @@ import WizardFooter, {InheritedWizardFooterProps, WizardFooterProps} from "./Wiz
  * @param {size} indicates the size of the wizard component
  * @param {style} custom inline CSS styles for the wizard component
  * @param {title} title React component to use for the wizard
- *
+ * @param {isApplying} if true then disable navigation and footer and show loading spinner on complete button
```

Comment:
As suggested above, can we rename it to be little generic to cater other usecases too.

Reply:
Take care of this at https://github.com/EMCECS/clarity-react/pull/143/